learn/os_setup/install_gfortran.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: book
+title: Installing GFortran
+permalink: /learn/os_setup/install_gfortran
+---
+
+GFortran is the name of the [GNU Fortran project](https://gcc.gnu.org/fortran/). The main [wiki](https://gcc.gnu.org/wiki/GFortran) page offers many helpful links about GFortran, as well as Fortran in general. In this guide, the installation process for GFortran on Windows, Linux, and macOS is presented in a beginner-friendly format based on the information from [GFortranBinaries](https://gcc.gnu.org/wiki/GFortranBinaries).
+
+## Windows
+
+
+Three sources provide quick and easy way to install GFortran compiler on Windows:
+1. [http://www.equation.com](http://www.equation.com/servlet/equation.cmd?fa=fortran), provides 32 and 64-bit x86
+ executables of the latest (10.2) gcc-version.
+2. [TDM GCC](https://jmeubank.github.io/tdm-gcc/articles/2020-03/9.2.0-release), provides 32 and 64-bit x86 executables of the 9.2 gcc-version.
+3. [MinGW-w64](http://mingw-w64.org/doku.php/download/mingw-builds) provides a 64-bit x86 executable of the 7.2 gcc-version.
```

Comment:
```suggestion
3. [MinGW-w64](http://mingw-w64.org/doku.php/download/mingw-builds) provides a 64-bit x86 executable of the 8.1.0 gcc-version.
```

I've downloaded this particular distribution and can confirm that it contains `gcc` version `8.1.0`. (The associated download page has the wrong version listed.)